components/prospects/CreateProspectNote.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react'
+import { Box, Button, TextField } from '@mui/material'
+import CancelIcon from '@mui/icons-material/Cancel'
+import CreateIcon from '@mui/icons-material/Create'
+import SaveIcon from '@mui/icons-material/Save'
+
+function CreateProspectNote({ prospect }) {
+  const [prospectNote, setProspectNote] = useState('')
+  const [isEditing, setIsEditing] = useState(false)
+
+  const handleProspectNoteChange = (event) => {
+    setProspectNote(event.target.value)
+  }
+
+  const handleCancelNote = () => {
+    setIsEditing(false)
+    setProspectNote('')
+  }
+
+  return (
+    <>
+      <Box style={{
+        display: 'flex',
+        justifyContent: 'flex-end',
+        alignItems: 'center'
+      }}>
+        {!isEditing && (
+          <Button style={{
+            textTransform: 'none',
```

Comment:
do we need this css or can we just use <Button>?

Reply:
I think the button looks better with 'textTransform'.
What is your opinion?
Do you think it would be better if there were only capital letters?